src/main/resources/frontend/src/services/adminService.ts

```diff
@@ -100,9 +100,9 @@ export const getQueueStatus = async (): Promise<QueueStatusResponse> => {
     }
 }
 
-export const commitAnalyticsGet = async (): Promise<string> => {
+export const commitAnalyticsGet = async (option: string): Promise<string> => {
```

Comment:
Could you add a Typescript definition containing the possible values for option?